README.md

```diff
@@ -1,4 +1,4 @@
-# Conduit Connector for <resource>
+# Conduit Connector for MySQL
 
 [Conduit](https://conduit.io) connector for <resource>.
```

Comment:
We need to include MySQL on `<resource>`

Reply:
Below, we need to make sure we also include the added configuration (host and port)